src/api/tests/orders/get-api-orders-id.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { test, expect } from 'fixtures/index';
+import { STATUS_CODES } from 'data/status.code';
+import { ICreateOrdersData } from 'types/orders.type';
+import { validateSchema } from 'utils/notifications/validations/schemaValidation';
+import { orderSchemaResponse } from 'data/schemas/orders/order.schema';
+import { faker } from '@faker-js/faker';
+import { ERRORS } from 'data/errorMessages';
+import { baseSchema } from 'data/schemas/base.schema';
+
+async function createOrderData(
+  customerService: any,
+  productService: any,
+  token?: any,
+  numberOFProducts: number = 1,
+) {
+  if (typeof numberOFProducts !== 'number') {
+    throw new Error('Incorrect type of Products');
+  }
+  const customer = await customerService.create(token);
+  const product = await productService.create(token);
+  const orderData: ICreateOrdersData = {
+    customer: customer._id,
+    products: [],
+  };
+
+  for (let i = 1; i <= numberOFProducts; i++) {
+    orderData.products.push(product._id);
+  }
+  return orderData;
+}
+
+test.describe('[API], [Orders] GET /api/orders/{id}', () => {
+  let token: string;
+  let orderId: string;
+
+  test.beforeAll(async ({ signInService }) => {
+    token = await signInService.loginAsLocalUser();
+  });
+
+  test.afterAll(async ({ orderService }) => {
+    if (orderId) {
+      await orderService.fullDelete(orderId, token);
+    }
+  });
+
+  test('Should get correct response, with the valid productId in query ', async ({
```

Comment:
И так же ниже поправить `productId > orderId`
```suggestion
  test('Should get order by order id, async ({
```